ROS/osr_bringup/launch/osr.launch

```diff
@@ -11,9 +11,9 @@
             <param name="enable_turbo_button" value="5"/>  <!-- -1: disable turbo -->
             <param name="axis_linear" value="1"/>  <!-- which joystick axis to use for driving -->
             <param name="axis_angular" value="3"/>  <!-- which joystick axis to use for turning -->
-            <param name="scale_linear" value="0.24"/>  <!-- scale to apply to drive speed, in m/s: drive_motor_rpm * 2pi / 60 * wheel radius -->
-            <param name="scale_angular" value="0.53"/>  <!-- scale to apply to angular speed, in rad/s: scale_linear / min_radius -->
-            <param name="scale_linear_turbo" value="0.5"/>  <!-- scale to apply to linear speed, in m/s -->
+            <param name="scale_linear" value="0.8"/>  <!-- scale to apply to drive speed, in m/s: drive_motor_rpm * 2pi / 60 * wheel radius -->
+            <param name="scale_angular" value="1.75"/>  <!-- scale to apply to angular speed, in rad/s: scale_linear / min_radius -->
+            <param name="scale_linear_turbo" value="1.78"/>  <!-- scale to apply to linear speed, in m/s -->
```

Comment:
Haven't tested these myself, but guess they're good. 1.78 looks right per the calc in the readme (223*2*pi/60 * 0.075)